pkg/qcbor/Makefile

```diff
@@ -0,0 +1,9 @@
+PKG_NAME    = qcbor
+PKG_URL     = https://github.com/laurencelundblade/QCBOR
+PKG_VERSION = ee146ab6105344a18744b34c38bc1cf3e5135b02 # Commit on Feb 19, 2020
```

Comment:
I checked the upstream repo, the commit following this just moved the includes to a separate folder, so why not chose a more recent commit, adapt to that change so that following updates are simpler 😃

Reply:
no problem :) done